### PR TITLE
migrated more <img> to next/image

### PR DESCRIPTION
**Pull request by Keitio**

### CODE REVIEW

File: src/components/DailyInfo.tsx [hunk 1]
```diff
@@ -119,13 +129,11 @@ function DailyRow({ dayOfWeek }: DailyRowProps): JSX.Element {
 
           <div className="flex justify-center flex-wrap">
             {weaponUpgrades.map((weaponUpgrade, index) => (
-              <div key={index}>
-                <img
+              <div key={index} className="relative h-20 w-20">
```
Comment (KeziahMoselle) [inline src/components/DailyInfo.tsx:132]: Shouldn't the size be written as props on the Image component ? 

```js
<Image height={} width={} />
```

File: src/components/GuerillaTimers.tsx [hunk 2]
```diff
@@ -107,16 +92,17 @@ const TimerRow = ({ guerilla }): JSX.Element => {
...
         {guerillaTypes(endDate.getDay(), guerilla.start[1]).map(
           (weapon, index) => (
-            <img
-              src={weaponToIcon(weapon)}
-              alt={`${weapon} icon`}
-              title={weapon}
-              className="h-16"
-              key={index}
-            />
+            <div key={index} className="w-16 h-16 relative">
```
Comment (KeziahMoselle) [inline src/components/GuerillaTimers.tsx:98]: Same here ?

File: src/components/Layout/Header.tsx [hunk 3]
```diff
@@ -10,9 +12,13 @@ export default function Header(): JSX.Element {
     <header className="container">
       <div className="flex flex-col justify-center items-center flex-wrap gap-y-8 px-8 py-6">
         <Link href="/" passHref={true}>
-          <a className="logo flex items-center">
-            <img className="h-20 mr-4" src="/logo-256.png" alt="Logo" />
-            <h1 className="text-3xl text-center">NieR Re[in] Guide</h1>
+          <a className="flex items-center relative group">
+            <span className="h-20 w-20 mr-4 z-0 absolute -top-16 left-1/2 transform -translate-x-1/2 scale-100 group-hover:scale-110 transition-transform ease-out-cubic">
+              <Image className="" src={logoImage} alt="Logo" />
```
Comment (KeziahMoselle) [inline src/components/Layout/Header.tsx:17]: detail: empty `className`

**Review by KeziahMoselle — APPROVED** [review]
Great job !

File: src/components/DailyInfo.tsx (hunk 1 above)
Comment (Keitio) [reply src/components/DailyInfo.tsx:132]: Wouldn't work for responsive (only in px, not in rem)

File: src/components/Layout/Header.tsx (hunk 3 above)
Comment (Keitio) [reply src/components/Layout/Header.tsx:17]: Will fix

**Pull request merged**